tangram/src/lib.rs

```diff
@@ -0,0 +1,559 @@
+//
+//           _______________________________ ________
```

Comment:
Please, change this header with the new one (GameDAO). You can find examples in the other pallets.